subprojects/instant-execution/src/main/kotlin/org/gradle/instantexecution/DefaultInstantExecution.kt

```diff
@@ -81,6 +82,10 @@ class DefaultInstantExecution(
         host.newStateSerializer()
     }
 
+    private
+    val buildOperationExecutor: BuildOperationExecutor =
```

Comment:
❌ I'd make this a getter instead of a field to avoid the service query in non-instant execution runs.